spec/ValueHandler/MetricPropertyValueHandlerSpec.php

```diff
@@ -4,21 +4,22 @@
 
 namespace spec\Webgriffe\SyliusAkeneoPlugin\ValueHandler;
 
+use PhpSpec\ObjectBehavior;
 use Sylius\Component\Core\Model\ProductInterface;
 use Sylius\Component\Core\Model\ProductVariant;
 use Sylius\Component\Core\Model\ProductVariantInterface;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Webgriffe\SyliusAkeneoPlugin\Converter\UnitMeasurementValueConverterInterface;
 use Webgriffe\SyliusAkeneoPlugin\ValueHandler\MetricPropertyValueHandler;
-use PhpSpec\ObjectBehavior;
 use Webgriffe\SyliusAkeneoPlugin\ValueHandlerInterface;
 
 class MetricPropertyValueHandlerSpec extends ObjectBehavior
 {
     private const AKENEO_ATTRIBUTE_CODE = 'akeneo_attribute_code';
 
     private const PROPERTY_PATH = 'property_path';
-    private const KG_23_VALUE = [0 => ['data' => ['amount' => 23, 'unit' => 'KILOGRAM']]];
+
+    private const KG_23_VALUE = [0 => ['data' => ['amount' => '23.0000', 'unit' => 'KILOGRAM']]];
```

Comment:
The amount value format changed, is it correct?

Reply:
It's how the value is returned from Akeneo, but yes, it is not relevant for this PR...